operator/config/overlays/openshift/kustomization.yaml

```diff
@@ -34,16 +34,18 @@ configMapGenerator:
   - controller_manager_config.yaml
   name: manager-config
 
-patchesStrategicMerge:
-- auth_proxy_service_annotations_patch.yaml
-- manager_auth_proxy_patch.yaml
-- manager_related_image_patch.yaml
-- manager_run_flags_patch.yaml
-- manager_security_context_patch.yaml
-- manager_webhook_patch.yaml
-- prometheus_service_monitor_patch.yaml
 
 images:
 - name: controller
   newName: quay.io/openshift-logging/loki-operator
   newTag: 0.1.0
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+patches:
```

Comment:
Move to the top?

Reply:
This is a result of `kustomize edit --fix`. Fixing this for better readability as suggested.